#goal definition
bool on_off
float64[] transform_a_big_pulley
float64[] transform_a_small_pulley
float64[] transform_b_big_pulley
float64[] transform_b_small_pulley
---
#result definition
bool done
---
#feedback
bool sequence
